### ISSUE TITLE
docker import using --change value with double quotes throws `bare " in non-quoted-field` error

### ISSUE THREAD

**Issue by feedthefire** [docker/docker#27789]
**Description**

Running `docker import` using a `--change` value which contains a double quote fails with an error.

**Steps to reproduce the issue:**
1. Export a container into a tar file or pipe to STDOUT
2. Import the exported tar file (or STDIN) with a `--change` value which contains a double quote
3. Produces error `bare " in non-quoted-field`

**Describe the results you received:**
Running the following command:
```
docker import --change 'ENTRYPOINT ["/entrypoint.sh"]' --change 'EXPOSE 80' --change 'CMD ["runapp"]' test_export_2016-10-26.tar test_export:2016-10-26`
```
throws this error:
```
invalid argument "ENTRYPOINT [\"/entrypoint.sh\"]" for --change: line 1, column 12: bare " in non-quoted-field
```

**Describe the results you expected:**
Running `docker import` with a `--change` value which contains double quotes runs without error.

**Additional information you deem important (e.g. issue happens only occasionally):**
Re-filing from this issue: https://github.com/docker/for-mac/issues/108 

**Output of `docker version`:**

```
Docker version 1.12.1, build 6f9534c
```

**Output of `docker info`:**

```
Containers: 12
 Running: 7
 Paused: 0
 Stopped: 5
Images: 390
Server Version: 1.12.1
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 341
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: null bridge host overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Security Options: seccomp
Kernel Version: 4.4.20-moby
Operating System: Alpine Linux v3.4
OSType: linux
Architecture: x86_64
CPUs: 4
Total Memory: 3.855 GiB
Name: moby
ID: YPKP:LTNP:73C6:FOPB:P4OB:UT7C:VLL3:BAUU:UZOU:FYDD:HCGR:4LAT
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): true
 File Descriptors: 46
 Goroutines: 72
 System Time: 2016-10-26T16:32:07.318518229Z
 EventsListeners: 1
No Proxy: *.local, 169.254/16
Registry: https://index.docker.io/v1/
Insecure Registries:
 127.0.0.0/8
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
None

**Comment by yongtang** [comment]
I think this one has been fixed (See #26173) in the master some time ago.

**Comment by thaJeztah** [comment]
Yes, looks like this was resolved, but didn't find its way into 1.12.2/1.12.3. This will be fixed in the next (1.13) release.

I'll close this issue, because this is already fixed, but thanks for reporting @feedthefire !
